edx_exams/apps/api/v1/views.py

```diff
@@ -565,6 +565,8 @@ class CourseExamAttemptView(ExamsAPIView):
             }
     """
 
+    authentication_classes = (JwtAuthentication,)
```

Comment:
I think this need to be paired with a permission class (minimally IsAuthenticated) to work